osbs/api.py

```diff
@@ -294,6 +294,13 @@ def _create_scratch_build(self, build_request):
         build_json['spec']['serviceAccount'] = 'builder'
         build_json['metadata']['labels']['scratch'] = 'true'
 
+        if build_request.low_priority_node_selector:
+            nodeselector = {}
+            for item in build_request.low_priority_node_selector.split(','):
```

Comment:
Can't we do the string manipulation in only in conf.py? and assume that at the api level it should already be a list?